projects/aca-content/src/lib/components/info-drawer/comments-tab/comments-tab.component.spec.ts

```diff
@@ -107,4 +113,30 @@ describe('CommentsTabComponent', () => {
       expect(checked).toContain('update');
     });
   });
+
+  describe('ExternalNodePermissionCommentsTabService', () => {
+    it('should return false if node permissions are correct and external service is not allowing it', async () => {
```

Comment:
can you rename title of test a bit because in expect you don't check directly result of service, instead you check canUpdateNode so please change a bit title to describe what is checked?